modules/nf-core/jvarkit/sam2tsv/meta.yml

```diff
@@ -0,0 +1,78 @@
+name: "jvarkit_sam2tsv"
+description: Convert sam files to tsv files
+keywords:
+  - sam
+  - tsv
+  - jvarkit
+tools:
+  - "jvarkit":
+      description: "Java utilities for Bioinformatics."
+      homepage: "https://github.com/lindenb/jvarkit"
+      documentation: "https://jvarkit.readthedocs.io/"
+      tool_dev_url: "https://github.com/lindenb/jvarkit"
+      doi: "10.6084/m9.figshare.1425030"
+      licence: ["MIT License"]
+      identifier: ""
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test' ]
+    - bam:
+        type: file
+        description: BAM/CRAM file from alignment
+        pattern: "*.{bam,cram}"
+    - bai:
+        type: file
+        description: BAI/CRAI file from alignment
+        pattern: "*.{bai,crai}"
+  - - meta2:
+      type: map
+      description: |
+        Groovy Map containing VCF information
+    - fasta:
+        type: file
+        description: Input fasta file
+        pattern: "*.{fasta,fa}"
+    - fasta_index:
+        type: file
+        description: Reference genome information for fasta index
+        pattern: "*.{fasta.fai,fa.fai}"
+    - fasta_dict:
+        type: file
+        description: Reference genome information for fasta dict
+        pattern: "*.{.dict}"
+  - - meta3:
+        type: map
+        description: |
+          Groovy Map containing reference fai information
+          e.g. [ id:'test_reference' ]
```

Comment:
This description seems to belong to something else. meta3 contains the metadata of the regions_file